index.d.ts

```diff
@@ -115,6 +115,9 @@ declare namespace WAWebJS {
         /** Searches for messages */
         searchMessages(query: string, options?: { chatId?: string, page?: number, limit?: number }): Promise<Message[]>
 
+        /** Download message media for cached messages only */
+        downloadMedia(mediaInfo: MediaInfo): Promise<MessageMedia>
```

Comment:
```suggestion
        downloadMedia(): Promise<MessageMedia>
```

Reply:
Why you removed the `mediaInfo` and how you are accessing it like `this.mediaInfo` if you look it carefully this `downloadMedia()` method is in `Client.js` which does not contain any information about messages.

Reply:
> Why you removed the `mediaInfo` and how you are accessing it like `this.mediaInfo` if you look it carefully this `downloadMedia()` method is in `Client.js` which does not contain any information about messages.

huh? let me re-review